english/snuffbox/explore/explore.md

```diff
@@ -0,0 +1,5 @@
+After the titanic sneeze rocked baseball sized holes in the marshmallow walls surrounding you. A curious thought enters your head.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.